README.md

```diff
@@ -33,54 +33,54 @@ The following table summarises currently supported vector extensions, compilers
 <table>
 <tr>
   <th colspan="2" rowspan="2"></th>
-  <th colspan="8">OS/Compiler</th>
+  <th colspan="9">OS/Compiler</th>
 </tr>
 <tr>
   <th colspan="3">Linux</th>
   <th colspan="2">MacOS</th>
-  <th colspan="3">Windows</th>
+  <th colspan="4">Windows</th>
 </tr>
 <tr>
   <th>Arch.</th>
   <th>Vector Extensions</th>
   <th>gcc</th><th>llvm</th><th>icc</th>
   <th>gcc</th><th>llvm</th>
-  <th>gcc</th><th>llvm</th><th>msvc</th>
+  <th>gcc</th><th>llvm-gnu</th><th>llvm-msvc</th><th>msvc</th>
 </tr>
 <tr align="center"><th>x86_64</th><th>SSE2, SSE4,<br>AVX, AVX2, AVX512F</th>
   <td>:green_circle:</td><td>:green_circle:</td><td>:white_circle:</td>
   <td>:white_circle:</td><td>:white_circle:</td>
-  <td>:white_circle:</td><td>:white_circle:</td><td>:white_circle:</td>
+  <td>:white_circle:</td><td>:green_circle:</td><td>:white_circle:</td><td>:white_circle:</td>
```

Comment:
Thanks for taking the time to document that.
In this case only DFT is tested though. What we do at the moment is at least add a comment in the following subsections, here `Component support` section should indicate that on Windows only DFT is tested but everything is built.

Let's not bother in this patch but we should probably use a different colour scheme when only part of the library is tested, or when component is compiled only and not tested. Ultimately, this could probably be mapped to the workflow/job status.